src/components/sketches/test4.ts

```diff
@@ -0,0 +1,110 @@
+import p5 from 'p5';
+export default function sketch(p: p5): void {
+  let canvas: p5.Renderer;
+
+  const particles: Particle[] = [];
+  const windowWidth = window.innerWidth;
+  const windowHeight = window.innerHeight;
```

Comment:
i thinkj we should just remove these since we want to do a more responsive model anyways